sklearn/tests/test_multioutput.py

```diff
@@ -561,3 +561,30 @@ class A(MultiOutputEstimator, MultiOutputRegressor):
 
     with pytest.warns(FutureWarning, match="is deprecated in version 0.22"):
         A(SGDRegressor(random_state=0, max_iter=5))
+
+
+class GradientBoostingRegressorTestFitParam(GradientBoostingRegressor):
+    def fit(self, X, y, sample_weight=None, monitor=None, **fit_params):
+        self.fit_params = fit_params
+        super().fit(X, y, sample_weight=sample_weight)
+
+
+def test_test_multi_target_regression_with_fit_params():
+    X, y = datasets.make_regression(n_targets=3)
+    X_train, y_train = X[:50], y[:50]
```

Comment:
I don't think that we care about `prediction` here. `fit_params` will only be affecting `fit`. No need for splitting.